universal-login-relayer/test/helpers/startRelayer.js

```diff
@@ -35,7 +36,19 @@ async function startRelayer(wallet, relayerConstructor) {
       {
         address: tokenContract.address,
         minimalAmount: utils.parseEther('0.5').toString()
-    }]
+    }],
+    localization: {
+      language: 'en',
+      country: 'any'
+    },
+    onRampProviders: {
+      safello: {
+        appId: '1234-5678',
+        baseAddress: 'https://app.s4f3.io/sdk/quickbuy.html',
+        addressHelper: true
+      }
+    },
+    database: getKnexConfig(),
```

Comment:
There is apparent duplication between `startRelayer` and `createPreconfiguredRelayer`. Can we get rid of it?